components/feature/session/src/main/java/mozilla/components/feature/session/PictureInPictureFeature.kt

```diff
@@ -11,22 +11,28 @@ import android.os.Build
 import androidx.annotation.RequiresApi
 import mozilla.components.browser.session.SessionManager
 import mozilla.components.browser.session.runWithSessionIdOrSelected
+import mozilla.components.support.base.crash.CrashReporting
+import mozilla.components.support.base.log.logger.Logger
 
 /**
  * A simple implementation of Picture-in-picture mode if on a supported platform.
  *
  * @param sessionManager Session Manager for observing the selected session's fullscreen mode changes.
  * @param activity the activity with the EngineView for calling PIP mode when required; the AndroidX Fragment
  * doesn't support this.
+ * @param crashReporting Instance of `CrashReporting` to record unexpected caught exceptions
  * @param customTabSessionId ID of custom tab session.
  * @param pipChanged a change listener that allows the calling app to perform changes based on PIP mode.
  */
 class PictureInPictureFeature(
     private val sessionManager: SessionManager,
     private val activity: Activity,
+    private val crashReporting: CrashReporting,
```

Comment:
This is a breaking API that would need to be fixed on the next snapshots update.

```kotlin
private val crashReporting: CrashReporting? = null
```

If you want to that update to the feature your own pace, a default nullable works too.